model_compression_toolkit/core/keras/pruning/check_node_role.py

```diff
@@ -0,0 +1,63 @@
+import keras.layers
+
+from model_compression_toolkit.core.common.framework_info import FrameworkInfo
+from model_compression_toolkit.core.common import BaseNode
+from model_compression_toolkit.core.common.pruning.pruning_section import PruningSection
+
+
+def is_keras_entry_node(node: BaseNode):
```

Comment:
Wouldn't it be simpler to put these methods in the keras_pruning_implementation? why do we need a separate file?